docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -227,6 +231,24 @@ worth considering no matter which statistic is showing high severity.
 (high-path-count)=
 ### Dealing with a high path count
 
+The number of control flow paths is a major indication of how difficult a rule
```

Comment:
we assume there is knowledge of what we call the control flow path. Most programmers think if it differently - without unrolling and without inlining, so our number look huge to them. Not sure however how easy is it to example, maybe we should have a webinar and then link to it

Reply:
rewrote things a little, added a sub-section with some explanations for the path count stat
 - mention that the CFG considered is the one with everything inlined
 - added a little picture with the diamond shapes, illustrating the exponential growth 
 - mention dispatcher summaries as an source of "diamonds"